Source/Core/Core/PowerPC/JitArm64/JitArm64_RegCache.h

```diff
@@ -115,6 +115,7 @@ class HostReg
   bool m_locked;
 };
 
+template <size_t GUEST_REG_COUNT>
```

Comment:
I wonder why you've choosen a template instead of a runtime parameter. std::vector isn't that slow ;)

But I don't care